src/components/Authorship.vue

```diff
@@ -0,0 +1,98 @@
+          <a v-bind:href="author.profile_link" target="_blank" v-text="author.fullName"></a>
+          <span v-if="index != Object.keys(additionalAuthors).length - 1 && Object.keys(additionalAuthors).length > 2">, </span>
+          <span v-if="index == Object.keys(additionalAuthors).length - 2"> and </span>
+        </span>
+        <span>
+        also contributed to the site.
+        </span>
+      </span>
+      <span id="contribution-statements" v-if="showContributionStatements">
+        <span id="primary-author-contribution">
+          <span
+            v-for="author in primaryAuthors" 
+            :key="`${author.initials}-contribution`"
+            :id="`author-${author.initials}`"
+            :class="'author'"
+          >
+            <span v-text="author.firstName"> </span> <span v-text="author.contribution"></span>. 
+          </span>
+        </span>
+        <span id="additional-author-contribution"  v-if="showAditionalContributionStatement">
+          <span
+            v-for="author in additionalAuthors" 
+            :key="`${author.initials}-contribution`"
+            :id="`author-${author.initials}`"
+            :class="'author'"
+          >
+            <span v-text="author.firstName"> </span> <span v-text="author.contribution"></span>. 
+          </span>
+        </span>
+      </span>
+    </p>
+  </div>
+</template>
+
+<script>
+import { isMobile } from 'mobile-device-detect';
+import authors from "@/assets/text/authors";
+export default {
+  name: "authorship",
+    components: {
+    },
+    props: {
+    },
+    data() {
+      return {
+        publicPath: process.env.BASE_URL, // allows the application to find the files when on different deployment roots
+        mobileView: isMobile, // test for mobile
+        primaryAuthors: authors.primaryAuthors,
+        additionalAuthors: authors.additionalAuthors,
+        showAdditionalAuthors: null, // Turn on or off attribution for additional authors
+        showContributionStatements: true, // Turn on or off contribution statements for ALL authors
+        showAditionalContributionStatement: null // if showContributionStatements is true, turn on or off contriubtion statements for ADDITIONAL authors
+      }
+    },
+    mounted(){   
+      this.showAdditionalAuthors =  this.additionalAuthors.length > 0 ? true : false; // Show author statements for additional authors if any are listed
+      this.showAditionalContributionStatement = this.additionalAuthors.length > 0 ? true : false; // Show contributions statements for additional authors if any are listed AND showContributionStatements is true
+    },
+    methods:{
+      isMobile() {
+              if(/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)) {
+                  return true
+              } else {
+                  return false
+              }
+          }
+    }
+}
+</script>
+<style>
+#author-container {
+  height: auto;
+  padding: 10px 0 10px 0;
```

Comment:
RC: Padding to the left and right as well as top and bottom. This text adjacent to window edge is hard to read and looks misaligned compared to the logo and/or footer: 
<img width="414" alt="image" src="https://user-images.githubusercontent.com/19958841/201708201-b04d34f9-e4db-49d9-a514-67019689f412.png">

`  padding: 10px 15px;` looks better: 

<img width="734" alt="image" src="https://user-images.githubusercontent.com/19958841/201710543-29473488-505d-4d84-8302-cf5c8d17067f.png">

Reply:
yeah agree, let's keep consistent with the other header/footer padding

Reply:
Agreed it needs padding when used as a footer. I left it out b/c I wasn't certain if this would be used as a footer, or inserted into the body of the page, like we often do with citations. I've added in the padding for now.